Things.md

```diff
@@ -169,4 +170,13 @@ Events can also be added to repeat, similar to using `setInterval`.  A new third
 FSM.addEventInterval(FSM.shiftHoriz, 100, 5, thing, 2);
 ```
 
-More can be read about TimeHandlr on its [Readme](https://github.com/FullScreenShenanigans/TimeHandlr/blob/master/README.md).
\ No newline at end of file
+More can be read about TimeHandlr on its [Readme](https://github.com/FullScreenShenanigans/TimeHandlr/blob/master/README.md).
+
+# Thing classes
+
+Classes are implemented using [ObjectMakr](https://github.com/FullScreenShenanigans/ObjectMakr), a module that automates the creation of the inheritance structure of classes. It takes in the structure and a list of properties and the class objects are then constructed using this information. 
```

Comment:
> Classes are implemented

1. Say classes are generated. "Implemented" is the wrong word (not technically accurate).